<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rampoven_Zone_Supr_ParClass</class>
 <widget class="QDialog" name="rampoven_Zone_Supr_ParClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rampoven_Zone_Supr_Par</string>
  </property>
  <widget class="QPushButton" name="ReadDataButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeDataButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>152</y>
     <width>217</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="ro_Highest_reading_Label">
        <property name="text">
         <string>Highest Reading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ro_Highest_Reading_SpinBox">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-3566.000000000000000</double>
        </property>
        <property name="maximum">
         <double>18030.000000000000000</double>
        </property>
        <property name="value">
         <double>-3566.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QDoubleSpinBox" name="ro_Lowest_Reading_SpinBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-3566.000000000000000</double>
        </property>
        <property name="maximum">
         <double>18030.000000000000000</double>
        </property>
        <property name="value">
         <double>18030.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ro_Lowest_Reading_Label">
        <property name="text">
         <string>Lowest Reading</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>141</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="ro_OutputTwo_Failsafe_Label">
      <property name="text">
       <string>Output 2:
Failsafe %</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ro_OutputTwo_Failsafe_SpinBox">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>147</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="ro_OutputOne_FailsafeLabel">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Output 1:&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Failsafe %&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ro_OutputOne_Failsafe_SpinBox">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>106</y>
     <width>176</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="ro_Loop_Break_Time_Label">
      <property name="text">
       <string>Loop Break
Time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="ro_Loop_Break_Time_SpinBox">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>3.000000000000000</double>
      </property>
      <property name="maximum">
       <double>9600.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>rampoven_Zone_Supr_ParClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>writeDataButton</sender>
   <signal>clicked()</signal>
   <receiver>rampoven_Zone_Supr_ParClass</receiver>
   <slot>WriteDataButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ReadDataButton</sender>
   <signal>clicked()</signal>
   <receiver>rampoven_Zone_Supr_ParClass</receiver>
   <slot>ReadDataButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>WriteDataButton()</slot>
  <slot>ReadDataButton()</slot>
 </slots>
</ui>
